<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QTableWidget" name="tblBuku">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>721</width>
     <height>331</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>119</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>No ISBN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Judul Buku</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penulis</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Penerbit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tahun Terbit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>harga</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblNoIsbn">
      <property name="text">
       <string>No ISBN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtnoIsbn"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lblJudulBuku">
      <property name="text">
       <string>Judul Buku</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtJudulBuku"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblPenulis">
      <property name="text">
       <string>Penulis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPenulis"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lblPenerbit">
      <property name="text">
       <string>Penerbit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPenerbit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="lblTahunTerbit">
      <property name="text">
       <string>Tahun Terbit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtTahunTerbit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="lblHarga">
      <property name="text">
       <string>Harga</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtPrice"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="lblCari">
      <property name="text">
       <string>Cari Judul Buku</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtSearch"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>239</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="btnSaveData">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEditData">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeleteData">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtnoIsbn</tabstop>
  <tabstop>txtJudulBuku</tabstop>
  <tabstop>txtPenulis</tabstop>
  <tabstop>txtPenerbit</tabstop>
  <tabstop>txtTahunTerbit</tabstop>
  <tabstop>txtPrice</tabstop>
  <tabstop>txtSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnSaveData</tabstop>
  <tabstop>btnEditData</tabstop>
  <tabstop>btnDeleteData</tabstop>
  <tabstop>tblBuku</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
